src/js/main-section.js

```diff
@@ -0,0 +1,73 @@
+import Notiflix from 'notiflix';
+import { Loading } from 'notiflix/build/notiflix-loading-aio';
+
+import { fetchBooks } from './fetchBooks.js';
+import Notiflix from 'notiflix';
+
+allCategoryBtn.addEventListener('click', onAllCategoriesClick);
+
+function onAllCategoriesClick() {
+  startPage();
+}
+
+startPage();
+
+function markupBooks(array) {
+  let bestBooks;
+  const categories = array
+    .map(arr => {
+      bestBooks = arr.books
+        .map(({ _id, book_image, title, author }) => {
+          let placeholder;
+          const imgSrc = true ? `${book_image}` : placeholder;
```

Comment:
this equation will always return true, therefore the value always will be "book_image" everything that stands after ":" will never be executed